incubating/service-now/step.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
   tags: []
   icon:
     type: svg
-    url: https://raw.githubusercontent.com/codefresh-io/steps/master/incubating/service-now/images/servicenow-1.png
```

Comment:
I think this will need to be 'master' once its published, right?

Reply:
yes